lib/browser/chrome-extension.js

```diff
@@ -309,7 +314,11 @@ app.once('ready', function () {
     const manifest = manifestMap[parsed.hostname]
     if (!manifest) return callback()
 
-    if (parsed.path === '/_generated_background_page.html' &&
+    const backgroundPath = '/' +
+      (manifest.background && manifest.background.page ||
```

Comment:
Can you save the name of background page to `backgroundPages` so we don't have to test `background.page` in all places?